### PR TITLE
Update versions to latest in csv

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
Signed-off-by: Puneet Punamiya ppunamiy@redhat.com

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml [hunk 1]
```diff
@@ -278,8 +278,8 @@ spec:
     ### Components
 
     - Tekton Pipelines: v0.47.0
```
Comment (vdemeester) [inline operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml:280]: This will change "soon-ish" 😅

Comment (vdemeester) [reply operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml:280]: https://github.com/tektoncd/pipeline/releases/tag/v0.47.2

Comment (vdemeester) [reply operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml:280]: And this needs to be updated in `release-v0.67.x` 👼🏼

**Review by vdemeester — CHANGES_REQUESTED** [review]
Also see https://github.com/tektoncd/operator/pull/1563

**Comment by jkandasa** [comment]
/hold
please resolve comments from @vdemeester

**Pull request closed without merging**